eval/unwrap.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/elves/elvish/eval/vals"
 )
 
+const (
```

Comment:
These are not exported symbols, so don't give them names starting with a capital letter.

They should be only be used in one place and are very common names for the 3 standard files, so it's simpler to remove the constants and use string literals in the functions below,